lib/LoginStrategy.ts

```diff
@@ -105,6 +100,17 @@ export class LoginStrategy extends BaseStrategy {
     this.jwtSignOptions = jwtSignOptions
     this.privateKey = privateKey!
   }
+
+  private async initCacheServer(privateKey: string, cacheServerUrl: string) {
+    const cacheServerClientInstance = new CacheServerClient({
+      privateKey,
+      provider: this.provider,
+      url: cacheServerUrl,
+    });
+    await cacheServerClientInstance.login();
+    this.cacheServerClient = cacheServerClientInstance;
```

Comment:
@jrhender The key part

Reply:
Ahhhh, I see. The key is the assignment of `this.cacheServerClient` after `login()` is completed.

Reply:
Yeah, that is the point